Scripts/Abilities/MysticArc.cs

```diff
@@ -46,7 +46,7 @@ public override void OnHit(Mobile attacker, Mobile defender, int damage)
 
             if (m_Target != null)
             {
-                defender.MovingEffect(m_Target, weapon.ItemID, 18, 1, false, false);
+                defender?.MovingEffect(m_Target, weapon.ItemID, 18, 1, false, false);
```

Comment:
Why the ? 

Do we need that there?

Reply:
![image](https://user-images.githubusercontent.com/17765272/80003888-51de3a00-84ca-11ea-8998-b50de45c7359.png)

Reply:
Is that a style change we want to use? We do not use the ? anywhere else to != null. I say we do all or none with the style changes.

Reply:
According to this: https://stackoverflow.com/questions/40992457/c-sharp-6-0-multiple-identical-null-conditional-operator-checks-vs-single-tradit

There is very little to no performance difference. Also stated that using the traditional null checks keep the code blocks nice and organized.

Reply:
It's not really a style, just a shortcut operator. It's like using ?? or the short conditional ?.

Reply:
I use a lot of Reshapers suggestions BUT I have noticed it sometimes is not right or relevant to our project. I love Reshaper but some stuff I do ignore.